### PR TITLE
Animas annotations

### PR DESCRIPTION
**Pull request by gniezen**
See https://trello.com/c/eTd9Ulgi for details.

### CODE REVIEW

**Comment by gniezen** [comment]
Pinging @jh-bate for yet another small code review. If this one passes, will you merge and bump tideline version for Blip on integration environment to enable these annotations for alpha?

**Comment by gniezen** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;eTd9Ulgi&#x2F;14-tideline-add-support-for-animas-specific-annotations"> [tideline] Add support for Animas-specific annotations</a>

**Comment by gniezen** [comment]
As discussed during all-hands: @jebeck - if you're happy to review this and merge and bump tideline version for Blip on integration environment to enable these annotations for alpha, please go ahead :)

File: js/plot/util/annotations/annotationdefinitions.js [hunk 1]
```diff
@@ -34,6 +34,27 @@ var definitions = {
       var b = '\'s threshold for reporting a numerical value. Your actual BG value was higher or lower than displayed.';
       return defs.stitch(a, b, source);
     },
+    'animas/bolus/extended-equal-split': function(source, defs) {
```
Comment (jebeck) [inline js/plot/util/annotations/annotationdefinitions.js]: I looked into why these aren't showing up when embedded in wizards. The filtering is fine, the problem is that we call a method to draw annotations for wizards and then we call a method to draw annotations for non-wizard boluses, and the latter overwrite the former. Not going down the rabbithole to try to tease that apart right now 😕

**Comment by jebeck** [comment]
Annotations are 👍 

I'll merge and bump version on blip but hold off on deployments for now with it being weekend down under and Darin away.

**Pull request merged**